packages/manager/modules/support/src/tickets/new-ticket/new-ticket.service.js

```diff
@@ -29,6 +31,22 @@ export const definition = class SupportNewTicketService {
   getSupportLevel() {
     return this.OvhApiMe.v6().supportLevel().$promise;
   }
+
+  createTicket(issue, subject, serviceName, urgency) {
+    let body = '';
```

Comment:
```suggestion
    const body = `${issue.subject}\n
    
    ${join(
      map(
        issue.fields,
        (field) => `${field.label}\n${field.default}\n`
      ),
      '\n',
    )}
    \n
    `;
```

don't think this is necessarely better though